opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -304,6 +305,31 @@ def format_span_id(span_id: int) -> str:
     return "0x{:016x}".format(span_id)
 
 
+def generate_span_id() -> int:
```

Comment:
the w3c tracecontext spec calls for generating valid span contexts in the case where one cannot parse tracecontext headers. As such it was necessary to lift this code into the API.